tools/README.md

```diff
@@ -0,0 +1,12 @@
+
+Be sure to put the following in: `/etc/udev/rules.d/99-magfest.rules` and execute `sudo udevadm control --reload-rules && sudo udevadm trigger`
```

Comment:
Checking if/how this works on windows would be nice too.

Reply:
Will work on Windows, just not right now.